checker/src/main/java/org/checkerframework/checker/sqltainting/BCryptPasswordEncoder.astub

```diff
@@ -0,0 +1,14 @@
+package org.springframework.security.crypto.bcrypt;
```

Comment:
This pull request contains both a `sqltainting` and a `sqlquotes` checker.  Can the former be removed?